packages/app-staking/src/Account/index.tsx

```diff
@@ -69,6 +71,7 @@ class Account extends React.PureComponent<Props, State> {
         {this.renderBondExtra()}
         {this.renderNominating()}
         {this.renderSessionKey()}
+        {this.renderUnbond()}
```

Comment:
Rather do `{this.state.isUnbondOpen  && this.renderUnbond()}` (or deconstruct this.state.isUnbondOpen before calling it).
 
There's no need to mount the `Unbond ` component, pass down the `isUnbondOpen` prop and then let the component return null if `isUnbondOpen` is false. It's better to just not call `this.renderUnbond()`  which mounts `<Unbond>`  at all if `isUnbondOpen` is false.

Same is true for all the other calls, so If you're up for refactoring tha in a second PR, feel free :).